tutorials/deploy-edge/production-deploy-to-ase-gpu.ipynb

```diff
@@ -0,0 +1,449 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Copyright (c) Microsoft Corporation. All rights reserved.\n",
+    "\n",
+    "Licensed under the MIT License."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Deploying a ML model as web service on Azure Stack\n",
+    "This notebook shows the steps to : registering a model, creating an image, provisioning,deploying a service using Iot Edge on Azure Edge. ![aml-flow](img/ml-flow-edge.png)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import azureml.core\n",
```

Comment:
remove this cell

Reply:
Why so? The notebook wouldn't work without it if I run it in the Jupyter server in Kubeflow.

Reply:
As per earlier comment, I'll leave the imports in the notebook. It will not hurt if the user uses requrements.txt, but it will not work if he does not.